.github/workflows/cmake.yaml

```diff
@@ -0,0 +1,31 @@
+name: cmake
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  cmake:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install Python prereqs
+        run: pip install pybind11[global]
+
+      - name: CMake configure
+        run: cmake -B ${{github.workspace}}/build
+
+      - name: CMake build
+        run: cmake --build ${{github.workspace}}/build --parallel `nproc`
+
+      - name: CMake test
+        working-directory: ${{github.workspace}}/build
+        run: ctest  --parallel `nproc`
```

Comment:
This test doesn't seem to do anything to test the python code. You are going to have to install pytest as I do in the other action and then run the tests to make sure it really does work from python.

Reply:
@timj Added python tests to the cmake GHA; please have another look?